usr/src/cmd/zfs/zfs_main.c

```diff
@@ -4152,7 +4156,7 @@ deleg_perm_type(zfs_deleg_note_t note)
 	}
 }
 
-static int inline
+inline static int
```

Comment:
I seriously doubt that any of these functions needs to be "inline".  Feel free to remove the inline while you're here, or we can leave it for another day.